### ISSUE TITLE
Should we create a file documenting all the feedback from our data holder? 

### ISSUE THREAD

**Issue by yangzhr** [SquareBoy/Good-team#5]
I only have notes documenting the feedback about the latest version of the leaflet

**Comment by dushujie** [comment]
Yes sure. now I  only have some handwritten notes too , It is inconvenient for our subsequent revision work. Maybe we should take the time to upload and integrate today or tomorrow？What do you think?

**Comment by dushujie** [comment]
please let me know when you guys are free~

**Comment by Xuchengchen** [comment]
I think it is a good note, I think we should first classify these feedback
.

**Comment by Xuchengchen** [comment]
I think we can sort by the following, including feedback on the flyer, feedback on the comics, and feedback on the data we choose.
